### PR TITLE
Helia and js-ipfs stragglers

### PR DESCRIPTION
**Pull request by ElPaisano**
Addresses @BigLep unaddressed comments on merged [PR 1620](https://github.com/ipfs/ipfs-docs/pull/1620). Comments are here https://github.com/ipfs/ipfs-docs/pull/1620#pullrequestreview-1530749239

### CODE REVIEW

**Comment by ElPaisano** [comment]
@BigLep I think I've addressed everything in the comments except the issue that https://docs.ipfs.tech/install/js-ipfs/#install-js-ipfs is still documenting js-ipfs and not Helia. I can use the wiki to create a "quickstart" that more or less mirrors the existing flow i.e. install, spawn a node, add a file, etc., what do think? Will change the file name and add redirects.

File: docs/concepts/faq.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ The quickest way to get IPFS up and running on your machine is by installing [IP
 
 For installing and initializing IPFS from the command line, check out the [command-line quick start](../how-to/command-line-quick-start.md) guide.
 
+### How do I learn more about IPFS standards and specifications?
+
+You can learn more about IPFS design standards and architectural specifications at [specs.ipfs.tech](https://specs.ipfs.tech/). The IPFS Standards website documents these standards and specifcations with the goal of fotering interoperability between independent implementations of the IPFS stack through Internet-grade specifications and test suites.
```
Comment (SgtPooki) [inline docs/concepts/faq.md:31]:
```suggestion
You can learn more about IPFS design standards and architectural specifications at [specs.ipfs.tech](https://specs.ipfs.tech/). The IPFS Standards website documents these standards and specifications with the goal of fostering interoperability between independent implementations of the IPFS stack through Internet-grade specifications and test suites. 
```

File: docs/concepts/faq.md [hunk 2]
```diff
@@ -49,6 +53,16 @@ There are already many IPFS implementations, including [Kubo](../install/command
 
 You can also create your own IPFS implementation. 
 
+### I am creating an implementation, how do I get started?
+
+If you would like to develop an IPFS implementation or are already working on one, the IPFS design standards and architectural specifications at [specs.ipfs.tech](https://specs.ipfs.tech/) are a great resource. In particular, the following resources are great starting points:
+
+- [IPFS Principles](https://specs.ipfs.tech/architecture/principles/), which provides context and details around the core IPFS principles of content-addressing and transport-agnosticity. The document defines what is or is not an IPFS implementation.
+- The [Meta](https://specs.ipfs.tech/meta/) section, which describes important non-technical information for contributors like what core project values, the governance model, how to produce documents, and more. 
+- [InterPlanetary Improvement Proposals (IPIPs)](https://specs.ipfs.tech/meta/ipip-process/) are a focused, transparent, communit-driven process for protocol design discussions. They are not changes to the specification itself, but their approval leads to a change in the specification.
```
Comment (SgtPooki) [inline docs/concepts/faq.md:62]:
```suggestion
- [InterPlanetary Improvement Proposals (IPIPs)](https://specs.ipfs.tech/meta/ipip-process/) are a focused, transparent, community-driven process for protocol design discussions. They are not changes to the specification itself, but their approval leads to a change in the specification.
```

File: docs/concepts/faq.md [hunk 3]
```diff
@@ -49,6 +53,16 @@ There are already many IPFS implementations, including [Kubo](../install/command
 
 You can also create your own IPFS implementation. 
 
+### I am creating an implementation, how do I get started?
+
+If you would like to develop an IPFS implementation or are already working on one, the IPFS design standards and architectural specifications at [specs.ipfs.tech](https://specs.ipfs.tech/) are a great resource. In particular, the following resources are great starting points:
```
Comment (SgtPooki) [inline docs/concepts/faq.md:58]:
```suggestion
If you want to develop an IPFS implementation or are already working on one, the IPFS design standards and architectural specifications at [specs.ipfs.tech](https://specs.ipfs.tech/) are a great resource. In particular, the following resources are great starting points:
```

File: docs/concepts/faq.md [hunk 4]
```diff
@@ -49,6 +53,16 @@ There are already many IPFS implementations, including [Kubo](../install/command
...
+
+- [IPFS Principles](https://specs.ipfs.tech/architecture/principles/), which provides context and details around the core IPFS principles of content-addressing and transport-agnosticity. The document defines what is or is not an IPFS implementation.
```
Comment (SgtPooki) [inline docs/concepts/faq.md:60]:
```suggestion
- [IPFS Principles](https://specs.ipfs.tech/architecture/principles/) provides context and details around the core IPFS principles of content-addressing and transport-agnosticism. The document defines what is or is not an IPFS implementation.
```

File: docs/concepts/faq.md [hunk 5]
```diff
@@ -49,6 +53,16 @@ There are already many IPFS implementations, including [Kubo](../install/command
...
+- The [Meta](https://specs.ipfs.tech/meta/) section, which describes important non-technical information for contributors like what core project values, the governance model, how to produce documents, and more.
```
Comment (SgtPooki) [inline docs/concepts/faq.md:61]:
```suggestion
- The [Meta](https://specs.ipfs.tech/meta/) section describes important non-technical information for contributors, like the core project values, the governance model, how to produce documents, and more. 
```

File: docs/concepts/faq.md [hunk 6]
```diff
@@ -49,6 +53,16 @@ There are already many IPFS implementations, including [Kubo](../install/command
...
+
+- [IPFS Principles](https://specs.ipfs.tech/architecture/principles/), which provides context and details around the core IPFS principles of content-addressing and transport-agnosticity. The document defines what is or is not an IPFS implementation.
+- The [Meta](https://specs.ipfs.tech/meta/) section, which describes important non-technical information for contributors like what core project values, the governance model, how to produce documents, and more. 
+- [InterPlanetary Improvement Proposals (IPIPs)](https://specs.ipfs.tech/meta/ipip-process/) are a focused, transparent, communit-driven process for protocol design discussions. They are not changes to the specification itself, but their approval leads to a change in the specification.
+
+In addition to these core documentats, `specs.ipfs.tech` documents standards for IPFS subsystems such as [InterPlanetary Naming System](https://specs.ipfs.tech/http-gateways/) and [HTTP Gateways](https://specs.ipfs.tech/http-gateways/).
```
Comment (SgtPooki) [inline docs/concepts/faq.md:64]:
```suggestion
In addition to these core documents, `specs.ipfs.tech` documents standards for IPFS subsystems such as the [InterPlanetary Naming System](https://specs.ipfs.tech/http-gateways/) and [HTTP Gateways](https://specs.ipfs.tech/http-gateways/).
```

File: docs/how-to/publish-ipns.md [hunk 7]
```diff
@@ -5,9 +5,10 @@ description: Learn how to publish IPNS names with Kubo
 
 # Publishing IPNS names
 
-IPNS names can be published programmatically.
+IPNS names can be published programmatically.
```
Comment (SgtPooki) [inline docs/how-to/publish-ipns.md:8]:
```suggestion
IPNS names can be published programmatically.
```

**Review by SgtPooki — CHANGES_REQUESTED** [review]

**Comment by SgtPooki** [comment]
> https://docs.ipfs.tech/install/js-ipfs/#install-js-ipfs is still documenting js-ipfs and not Helia. I can use the wiki to create a "quickstart" that more or less mirrors the existing flow i.e. install, spawn a node, add a file, etc., what do think? Will change the file name and add redirects.

Can we point users to helia-examples which will essentially be a living document?

**Comment by ElPaisano** [comment]
@SgtPooki thanks for getting to the GH action-flagged issues as well as the other issues you caught. The remaining GH action-flagged issues are one false positive for the link check and a vale flag for "IPIPs", so we can ignore. I also went through again and made some more changes. 

Can I have you pull this branch down and build the local preview? I'm trying to determine if there is a build error  that will cause issues in prod or if its just my local environment. Will explain more if you encounter any local build errors.

**Review by SgtPooki — APPROVED** [review]

**Review by BigLep — COMMENTED** [review]
Thanks for updating

**Pull request merged**